### PR TITLE
Corrected README for building go tools

### PR DESCRIPTION
**Pull request by derrickburns**
The README referred to a make target that does not exist.

@darinkrauss I am not certain that this is correct, but I think it is.  Please let me know.

### CODE REVIEW

**Comment by pazaan** [comment]
@derrickburns - you're right. Looks like it was removed in https://github.com/tidepool-org/platform/blame/53f758772091408c45d50d4019fcb8a4fa7d1305/Makefile#L91

File: README.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ For example:
 ```
 brew install go
 brew install dep
+brew install mongo
```
Comment (pazaan) [inline README.md]: I'm not sure if we want/need to prescribe how to run mongo. For example, I run mongo using a Docker container, with port 27018 poked through to localhost.

Comment (darinkrauss) [reply README.md]: I agree. I think the instructions above about installing and starting Mongo are good enough. We might want to specify the version, though. Production is currently 3.2 while I'm running 3.6.x locally.

Comment (darinkrauss) [reply README.md]: The same probably goes for the version of Go. How it is installed is irrelevant, I suppose.

OTOH, this is just an example, so maybe we should just leave it.

**Pull request merged**